aether-kernel/aether/kernel/api/serializers.py

```diff
@@ -168,13 +168,11 @@ class SubmissionSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
         queryset=models.Project.objects.all(),
         required=False,
     )
-    mappingset = MtPrimaryKeyRelatedField(
```

Comment:
You cannot remove this or multitenancy will not work properly :disappointed: you only need to add the option `required=False`